xess/xess.go

```diff
@@ -32,6 +32,8 @@ func init() {
 	URL = URL + "?cachebuster=" + anubis.Version
 }
 
+// Mount registers the xess static file handlers on the given mux
 func Mount(mux *http.ServeMux) {
-	mux.Handle("/.within.website/x/xess/", internal.UnchangingCache(http.StripPrefix("/.within.website/x/xess/", http.FileServerFS(Static))))
+	prefix := anubis.BasePrefix + "/.within.website/x/xess/"
```

Comment:
Can probably change this to `BasePrefix + /.xess`

Reply:
not sure what you mean. that would mean xess assets are served from /.xess. Seems odd to make an exclusion of assets to not be served from within `/.within.website/x/`